src/test/system_tests/test_utils.js

```diff
@@ -6,11 +6,13 @@ const _ = require('lodash');
 const http = require('http');
 const P = require('../../util/promise');
 const os_utils = require('../../util/os_utils');
+const nb_native = require('../../util/nb_native');
 const native_fs_utils = require('../../util/native_fs_utils');
 const config = require('../../../config');
 const { S3 } = require('@aws-sdk/client-s3');
 const { NodeHttpHandler } = require("@smithy/node-http-handler");
 const path = require('path');
+const { TYPES } = require('../../manage_nsfs/manage_nsfs_constants');
```

Comment:
I think it should be changed to `CONFIG_TYPES` (because the functions are abut the config file) and not add an import from the CLI.